src/apps/opening-hours-editor/useOpeningHoursEditor.tsx

```diff
@@ -21,9 +21,16 @@ const useOpeningHoursEditor = () => {
   const openingHoursBranchId = config("openingHoursBranchIdConfig", {
     transformer: "stringToNumber"
   });
+  const [datesSet, setDatseSet] = useState<null | DatesSetArg>(null);
   const queryClient = useQueryClient();
   const { data: openingHoursData } = useDplOpeningHoursListGET({
-    branch_id: openingHoursBranchId
+    branch_id: openingHoursBranchId,
+    ...(datesSet
+      ? {
+          from_date: getStringForDateInput(datesSet.start),
+          to_date: getStringForDateInput(datesSet.end)
+        }
+      : {})
```

Comment:
Consinder updating the Orval client to add an enabled property.

I appreciate your comment in the PR. You can see how we have added this for other requests in [`orval.config.js`](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/1a9f9df85eb133c1ff7583cbe60b0cdefa45d9bb/orval.config.ts#L132-L132).

If fetching all dates is actually a problem we should consider whether we want to mark from and to dates as required. That does not prevent callers from specifying from "1900-01-01 " and to "2100-01-01". Then we will end up with the same problem.